WordPress/Classes/ViewRelated/NUX/NoSitesView.swift

```diff
@@ -0,0 +1,63 @@
+import UIKit
+
+@objc protocol NoSitesViewDelegate {
+    func addSiteButtonPressed()
+}
+
+@objc class NoSitesView: UIView {
+
+    // MARK: - Properties
+
+    @objc weak var delegate: NoSitesViewDelegate?
+
+    @IBOutlet weak var noSitesTitle: UILabel!
+    @IBOutlet weak var addSiteButton: LoginButton!
+    @IBOutlet weak var titleLeadingConstraint: NSLayoutConstraint!
+    @IBOutlet weak var titleTrailingConstraint: NSLayoutConstraint!
+
+    // MARK: - Init
+
+    @objc class func instanceFromNib() -> NoSitesView {
```

Comment:
This feels like it should be a `UIViewController` and not a `UIView`. I know the `WPNoResultsView` it's replacing is a `UIView`, but it's also pure code layout. The `nib` makes it a bit more awkward imo. Also, child view controllers are awesome.

Not _telling_ you to make the change, however, but I'm curious to hear your thoughts 😀